packages/edit-post/src/components/header/style.scss

```diff
@@ -31,6 +31,53 @@
 	}
 }
 
+.is-distraction-free {
```

Comment:
Is there a reason why this appears at this point in the file? Would it be better for it to be at the end?